must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -59,5 +57,12 @@ for ns in $(oc get cephcluster --all-namespaces --no-headers | awk '{print $1}')
             COMMAND_OUTPUT_FILE=${COMMAND_OUTPUT_DIR}/${ceph_commands[$i]// /_}
             oc -n ${ns} exec -it ${operator_pod_name} -- ${ceph_commands[$i]} >> ${COMMAND_OUTPUT_FILE}
         done
+
+        COMMAND_JSON_OUTPUT_DIR=${CEPH_COLLLECTION_PATH}/namespaces/${ns}/must_gather_commands/json_output
```

Comment:
It's mostly a taste/style thing, but wouldn't it be a little nicer to capture the two different output formats of the same list of commands in one loop instead of looping twice?

Reply:
yup done